src/components/Table/Table.scss

```diff
@@ -64,6 +64,7 @@ $c-table-row-selected-bg: $color-primary !default;
 
 	td {
 		border-bottom: 0.1rem solid $c-table-border-color;
+		vertical-align: middle;
```

Comment:
Er is een `align` prop die `vertical-align: middle;` toepast op th en td elementen

Reply:
aha

Reply:
replaced by: https://github.com/viaacode/avo2-components/pull/260